### ISSUE TITLE
[Mod Request] Automessage

### ISSUE THREAD

**Issue by HonzaVinCZ** [ricksouth/serilum-mc-mods#1451]
## **Information**
Minecraft version: 1.18.2
Modloader: Forge

## **Feature description**
I would like to ask you for server side automessage mod with following features:
- Configurable ammount of messages ( Possibly array of messages which would allow any ammount of messages being sent )
- Configurable period when messages are sent ( Fixed period in seconds )
- Config option to randomize message order ( Randomize order true/false )
- Formating support
- Link support

Currently to this date, there are no purely automessage mods for Forge 1.18.2 and this could be a nice oppurtunity to fill this gap by a very useful server utility mod.
Basically, this mod could be very similar to Welcome Message but I'd like to see more open text formating options as we are used to ( If possible in newer MC versions, I still personally live partially in 1.12.2 era ).
Example:

> 	messages = ["&3This is &b&lAutoMessage &r&3mod", "&3Edit shown messages in &b&lAutoMessage &r&3config"]

**Comment by ricksouth** [comment]
I appreciate the new mod suggestion! I have added your idea to the list. I'm keeping track of all ideas on the Trello page and added the URL of this issue there.

In order to reduce the amount of open issues on this Github repository, I'll be closing this issue.

It definitely does not mean I am not interested and I'll post a comment here if/when the mod is released. This might take some time, as bug reports take priority. But I'll definitely be releasing more mods along the way.

Thank you very much for the suggestion!
